java/code.go

```diff
@@ -1,14 +1,15 @@
 package java
 
 import (
+	"github.com/zocdoc/gopoetry/util"
```

Comment:
Part of me thinks this import is wrong. Especially if we try to merge back to vlad's repo

Reply:
Yeah this is wrong. I can get this to work as just `gopoetry/util`. Something strange is afoot on your machine

Reply:
Ah cool. Thanks for confirming. I'll change them and see if I can compile on the commandline and ignore IDE redlines.